src/pykiso/test_coordinator/test_case.py

```diff
@@ -349,3 +349,22 @@ def func_wrapper(self: BasicTest) -> None:
         return func_wrapper
 
     return decorator
+
+
+def disable_logging(level: int = logging.CRITICAL):
```

Comment:
This would have expected to have this in logging_initializer

Reply:
Done